contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.3.0--2.4.0.sql

```diff
@@ -404,6 +404,97 @@ WHERE has_schema_privilege(s.schema_id, 'USAGE')
 AND c.contype = 'c' and c.conrelid != 0;
 GRANT SELECT ON sys.check_constraints TO PUBLIC;
 
+create or replace view sys.table_types_internal as
```

Comment:
did we make sure that a relevant upgrade test is in place for 14.6 to 14.7 upgrade?